### PR TITLE
Create kaseax.json

### PR DESCRIPTION
**Pull request by fynnbauer**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

Not online yet, only a reposerver & screenserver. Website will follow with the is-a-dev domain.

### CODE REVIEW

File: domains/kaseax.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "owner": {
+    "username": "Kaseax",
+    "email": "greenpepper1489@gmail.com",
+    "twitter": "kaseaxx",
+    "discord": "Kaseax#0864"
+  },
+  "record": {
+    "A": [
+      "37.114.32.204",
+      "37.114.32.205"
```
Comment (WilliamDavidHarrison) [inline domains/kaseax.json:11]:
```suggestion
      "37.114.32.205"
    ]
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/kaseax.json (hunk 1 above)
Comment (Kaseax) [reply domains/kaseax.json:11]: There are two different servers. why do you want that?

Comment (WilliamDavidHarrison) [reply domains/kaseax.json:11]: > There are two different servers. why do you want that?

You need the `]` to end the array...

Comment (Kaseax) [reply domains/kaseax.json:11]: oh, im sorry.

**Review by wdhdev — CHANGES_REQUESTED** [review]
You need to finish your website

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please finish your website

**Comment by WilliamDavidHarrison** [comment]
Will wait

**Comment by WilliamDavidHarrison** [comment]
Too long without a response

**Pull request closed without merging**